Extensions/blacklist.js

```diff
@@ -516,7 +516,7 @@ XKit.extensions.blacklist = new Object({
 				var m_author = "";
 				if (XKit.extensions.blacklist.preferences.check_authors.value) {
 					try {
-						var post_info_links = $(this).find(".post_info_link").map(function() {
+						var post_info_links = $(this).find(".post_info_link, .reblog-tumblelog-name.inactive").map(function() {
```

Comment:
always `.inactive`?

Reply:
Active blogs are included in .post_info_link already. Inactive ones aren't a link and don't have that class. Without inactive, some names will be captured twice, which I guess isn't a big deal either way.